lib/recurly/risk/three-d-secure/strategy/cybersource.js

```diff
@@ -16,9 +16,11 @@ export default class CybersourceStrategy extends ThreeDSecureStrategy {
     }
 
     const { PREFLIGHT_TIMEOUT } = CybersourceStrategy;
+    // debugger
     const data = {
       gatewayType: CybersourceStrategy.strategyName,
       gatewayCode: gateway_code,
+      billingInfoId: recurly.config.billingInfoId,
```

Comment:
IIRC we don't expect this to be set in the `recurly.config` but something in the `options` param of the element (adjacent to these style options [here](https://recurly.com/developers/reference/recurly-js/?#reference-1))